packages/create-app/templates/default-app/packages/backend/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN tar xzf skeleton.tar.gz && rm skeleton.tar.gz
 RUN yarn install --frozen-lockfile --production --network-timeout 300000 && rm -rf "$(yarn cache dir)"
 
 # Then copy the rest of the backend bundle, along with any other files we might want.
-COPY packages/backend/dist/bundle.tar.gz app-config.yaml ./
+COPY packages/backend/dist/bundle.tar.gz app-config*.yaml ./
```

Comment:
This will copy the `app-config.local.yaml` into the container when testing locally.

Reply:
nvm I should read